web-app/components/pages/login/login.jsx

```diff
@@ -24,117 +23,97 @@ const StyledGrid = styled(Grid)(() => ({
-                                },
-                            }}
-                            InputProps={{
-                                startAdornment: (
-                                    <InputAdornment position="start">
-                                        <LockRoundedIcon fontSize="medium" />
-                                    </InputAdornment>
-                                ),
-                            }}
-                        />
-                        <Button
-                            variant="contained"
-                            size="medium"
-                            color="primary"
-                            disableElevation
-                            sx={{ marginTop: '15px', textTransform: 'capitalize' }}
-                        >
-                            Login
-                        </Button>
-                        <Button
-                            variant="text"
-                            size="medium"
-                            color="success"
-                            disableElevation
-                            sx={{ color: 'grey', textTransform: 'capitalize' }}
-                        >
-                            Lupa Password?
-                        </Button>
-                        {/* <Box sx={{ fontFamily: 'serif' }}>
-                            <Typography component="span" sx={{ fontFamily: 'inherit' }}>
-                                Lupa&nbsp;
-                            </Typography>
-                            <Link href="#" underline="hover" sx={{ fontFamily: 'inherit', marginBottom: '20px' }}>
-                                {'Email / Password?'}
-                            </Link>
-                        </Box> */}
-                        <Typography component="span" fontSize="small" sx={{ top: '80px' }}>
-                            {'Belum punya akun?'}&nbsp;
-                            <Link component={RouterLink} to="/welcome" underline="hover" color="black">
-                                {'Daftar'}
-                            </Link>
-                        </Typography>
-                    </Box>
+                    <BoxImage>
+                        <img src="/images/logo-ira.png" alt="Logo" style={StyledImage} />
+                    </BoxImage>
+                    <form action="" method="post">
+                        <BoxLogin>
+                            <InputField
+                                label="Email"
+                                placeholder="Email"
+                                InputProps={{
+                                    startAdornment: (
+                                        <InputAdornment position="start">
+                                            <MailRoundedIcon fontSize="medium" />
+                                        </InputAdornment>
+                                    ),
+                                }}
+                            />
+                            <InputField
```

Comment:
Did you miss `type="password"` for this text field? This one is password field, isn't it?